go/libraries/doltcore/remotestorage/chunk_fetcher.go

```diff
@@ -263,6 +263,9 @@ func fetcherRPCDownloadLocsThread(ctx context.Context, reqCh chan *remotesapi.Ge
 			if err != nil {
 				return NewRpcError(err, "StreamDownloadLocations", host, stream.AssociatedReq())
 			}
+			if resp == nil {
```

Comment:
Seems weird that Recv can return a nil resp without an error... but this is better than a panic.